.gitignore

```diff
@@ -117,7 +117,8 @@ docs/formulation/*log
 /.werf_secret_key
 
 # Customized solver setup and docker files
-julia_src/xpress/licenseserver
+julia_src/xpress/
```

Comment:
this directory should no longer exist, correct? Is this a check just in case someone hasn't deleted it?

Reply:
Yes, it should no longer exist, but I'd rather keep this here as people transition over, just in case.